float32 center_x
float32 center_y
float32 w
float32 h
float32 confidence
float32 class_id
